src/sidebar/panel.js

```diff
@@ -155,101 +154,123 @@ function loadContent() {
             lastRemoteLoad = Date.now();
             if (!result.hasOwnProperty('content')) {
               console.log('No remote content. Loading local content.');
-              handleLocalContent(data);
+              return browser.storage.local
+                .set({ lastModified: result.last_modified })
+                .then(() => {
+                  return handleLocalContent(data);
+                });
             } else {
               console.log('Encrypted Content:', result['content']);
               return decrypt(keys, result['content'])
                 .then(content => {
                   handleConflictsMerge(
                     data.contentWasSynced,
                     data.notes,
-                    content
+                    content,
+                    result.last_modified
                   );
                 })
                 .catch(err => {
-                  handleLocalContent(data);
                   console.error(err);
+                  Promise.resolve(
+                    browser.storage.local
+                      .set({ lastModified: result.last_modified })
```

Comment:
In case we cannot decrypt we define the current local value as the remote value. We need to make sure here that the Bearer token is invalid in case the key changed.